.github/workflows/build-verification.yml

```
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
         matrix:
-            gradle-version: [ '6.0.1', '6.9.2', '7.0.2', '7.6', '8.0' ]
+            gradle-version: [ '6.0.1', '6.9.2', '7.0.2', '7.6.1', '8.0', '8.1.1' ]
```

These versions are all over the place. We should add a comment and/or update our process docs to explain when to add a new version vs. update an existing one and update this PR accordingly. The latest patch release of each is actually:

```suggestion
            gradle-version: [ '6.0.1', '6.9.4', '7.0.2', '7.6.1', '8.0.2', '8.1.1' ]
```

---

I committed the suggestion, good catches.

> But, is that actually what we want?

Our typical pattern is to test, for each major version we support:
- The latest release
- The latest patch release of the x.0.x minor version

This was aiming to meet that pattern. This action was not captured in our version update docs, so the versions were somewhat out of date.

---

+1 on the explanation by @clayburn